File: app/src/main/java/hu/wolfmanfp/timetable/courselist/CourseListViewHolder.java
package hu.wolfmanfp.timetable.courselist;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import hu.wolfmanfp.timetable.R;

public class CourseListViewHolder extends RecyclerView.ViewHolder {
    public TextView dateText;
    public TextView nameText;
    public TextView roomText;

    public CourseListViewHolder(View itemView) {
        super(itemView);
        dateText = itemView.findViewById(R.id.item_txtDate);
        nameText = itemView.findViewById(R.id.item_txtName);
        roomText = itemView.findViewById(R.id.item_txtRoom);
    }
}

File: app/src/main/java/hu/wolfmanfp/timetable/courselist/CourseListViewHolder.kt
package hu.wolfmanfp.timetable.courselist

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import hu.wolfmanfp.timetable.R

class CourseListViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    var dateText: TextView = itemView.findViewById(R.id.item_txtDate)
    var nameText: TextView = itemView.findViewById(R.id.item_txtName)
    var roomText: TextView = itemView.findViewById(R.id.item_txtRoom)

}
